docs/source/Appendix_B.md

```diff
@@ -154,6 +154,14 @@ An assembled event string must include all tags having the *required* schema att
 See [**3.2.10.2. Event-level processing**](./03_HED_formats.md#32103-event-level-processing) for
 additional information on the `required` tag.
 
+### SCHEMA_LOAD_FAILED

```

Comment:
Just to clarify, this will raise a HedFileError(which could potentially have SCHEMA_LOAD_FAILED as the error code)

Reply:
Yes.